adapters/webhdfs-adapter/src/main/java/com/connexta/replication/adapters/webhdfs/WebHdfsNodeAdapter.java

```diff
@@ -190,6 +221,10 @@ String getLocation(CreateStorageRequest createStorageRequest) throws URISyntaxEx
         .setParameter(HTTP_NO_REDIRECT_KEY, "true");
     HttpPut httpPut = new HttpPut(builder.build());
 
+    return handleLocationResponse(httpPut);
+  }
+
+  private String handleLocationResponse(HttpRequestBase httpRequest) {
```

Comment:
The method name should probably also describe the fact that it is sending the request as well. Maybe `handleLocationRequest`